### ISSUE TITLE
Patterns: Leveling-up WordPress.org Patterns

### ISSUE THREAD

**Issue by richtabor** [WordPress/gutenberg#49174]
This issue captures the current priorities for filling the WordPress.org _featured_ patterns within the Pattern Directory. 

Most of [these patterns](https://wordpress.org/patterns/author/wordpressdotorg/) are no longer bundled directly within WordPress core (https://github.com/WordPress/gutenberg/pull/46450), but instead pulled from the WordPress Pattern Directory—making it much quicker to improve/curate. 

I've split out tasks based on pattern creation, touching-up the Pattern Directory and improving the pattern experience.

### Pattern Categories: 

- [ ] [Banners](https://wordpress.org/patterns/author/wordpressdotorg/categories/banner/) — 6
- [ ] [Text](https://wordpress.org/patterns/author/wordpressdotorg/categories/text/) — (32 total, but 11 core patterns)
- [ ] [Posts](https://wordpress.org/patterns/author/wordpressdotorg/categories/query/) — 3
- [ ] [Call to Action](https://wordpress.org/patterns/author/wordpressdotorg/categories/call-to-action/) — 0
- [ ] Team — 0
- [ ] Testimonials — 0
- [ ] Services — 0
- [ ] Contact — 0
- [ ] About — 0
- [ ] Portfolio — 0
- [ ] [Gallery](https://wordpress.org/patterns/author/wordpressdotorg/categories/gallery/) — (9 total, but 3 core patterns)
- [ ] Media — 0
- [ ] [Header](https://wordpress.org/patterns/author/wordpressdotorg/categories/header/) — 8
- [ ] [Footer](https://wordpress.org/patterns/author/wordpressdotorg/categories/footer/) — 8

### Pattern Creation:

- [ ] https://github.com/WordPress/gutenberg/issues/47847
- [x] Conduct quality audit on current patterns
- [x] Ensure current patterns are in proper categories, now that we have new categories
- [ ] Add 20 patterns to each category

### Pattern Directory Tasks:

- [ ] Decide nomenclature for WordPress.org ("Curated") and Community patterns
- [ ] Enhance the WordPress Pattern Directory to show core patterns first ([related]( https://github.com/WordPress/pattern-directory/issues/395#issuecomment-1324319980))
- [ ] Add missing categories to the Pattern Directory (blocked by https://github.com/WordPress/gutenberg/issues/47847)
- [ ] Consider making all published WordPress.org authored patterns available in the Pattern Inserter

### Pattern Experience:

- [ ] Expose [wireframe](https://wordpress.org/patterns/author/wordpressdotorg/categories/wireframe/) patterns in the Pattern Inserter
- [x] https://github.com/WordPress/gutenberg/issues/49144
- [ ] https://github.com/WordPress/gutenberg/issues/48081
- [ ] https://github.com/WordPress/gutenberg/issues/44611

**Comment by cbirdsong** [comment]
I build custom themes at an agency, and we disable the pattern directory because they frequently include custom values for colors/font sizes/etc, which our themes do not allow. We would find it useful if the pattern directory was aware if the pattern uses custom values and we could configure the editor to only show patterns that do not include any. 

In the future, it would also be useful it the pattern directory could be filtered to only show patterns that use the standardized design tokens described in #39370, #39371, and #39372, but currently only spacing sizes are finalized.

(I don't know if this is the right place for this comment, but if an audit is occurring it seems like a good time to consider it. Let me know if it should be filed as a separate issue.)

**Comment by richtabor** [comment]
>  Decide nomenclature for WordPress.org (currently "Featured" or "Core") and Community patterns

Re this, I'd like to propose the top-level divide is "Curated Patterns" and "Community Patterns". 

I don't think "Featured" is a great term, as we have patterns that are added to a featured category and it will likely introduce confusion into the Pattern Explorer flow, with the top-level divide in that as well.
